tests/virt-handler_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors
+ *
+ */
+
+package tests_test
```

Comment:
This test was added as part of found regression and I marked it as such. I believe it is useful for the project but I agree it is not really e2e. I think we need to bring this up this discussion on how we want to group tests... but I like to keep it out of this PR

Reply:
The test may be useful to protect against a recurrence of a specific regression, but it is not a good test. It has a very strong dependency on implementation details and leaves out other leaks untested.

Splitting tests to sensible packages and owning teams urgently needed - but it can wait form another PR.

Reply:
>but it can wait form another PR.

Thanks